### PR TITLE
fix: cordova plugins inconsistent versions

### PR DESCRIPTION
**Pull request by b1zzu**
### Description

When building the ios or android app cordova will change the package.json.

#### How to reproduce

Clone the master in a new directory `/tmp/ionic-showcase-${RANDOM}`

execute this command with ionic@5 and cordova@9
```
npm install
ionic cordova build android
```
now check the directory
```
git status
```
> The `package.json` has been modified by cordova during the android build, same apply for ios

#### Changes
* Migrate all plugins from `config.xml` to `package.json` (require cordova > 9)
* Downgrade `cordova-support-google-services` because it is a dependency of `phonegap-plugin-push` and is the push plugin that decides the version 
* Add `cordova-plugin-app-version` because it is a dependency of `cordova-plugin-aerogear-metrics`
* Check that the `package.json` doesn't change after cordova build

##### Checklist

- [x] `npm test` passes
- [x] `npm run build` works
- [x] tests are included
- [x] documentation is changed or added

### CODE REVIEW

**Comment by wtrocki** [comment]
So if we merge - autoupdate mechanism will pick changes and bump packages again.
We can disable some updates in renovate.json if we want to monitor that manually.

**Comment by b1zzu** [comment]
> So if we merge - autoupdate mechanism will pick changes and bump packages again.
> We can disable some updates in renovate.json if we want to monitor that manually.

Yes, we should do it, but I don't know which are the packages managed by cordova and which not.

In cordova@9 plugins are stored in the package.json and not in the config.xml, so if we migrate all plugins to the package.json than renovate will work again correctly. 

But there is problem, in migrating to package.json, with all plugins which package name is different from the plugin name (like for the aerogear plugins)

**Comment by wtrocki** [comment]
So let's fix them for aerogear plugins. PRoblem is @aerogear/name that cannot be plugin name I think.

**Comment by b1zzu** [comment]
I'm trying to migrate all plugins to the package.json, let's see if it works correctly

File: package.json [hunk 1]
```diff
@@ -44,20 +44,23 @@
...
     "cordova-ios": "5.0.1",
+    "cordova-plugin-aerogear-metrics": "npm:@aerogear/cordova-plugin-aerogear-metrics",
+    "cordova-plugin-app-version": "0.1.9",
     "cordova-plugin-badge": "0.8.8",
     "cordova-plugin-device": "2.0.3",
     "cordova-plugin-dialogs": "2.0.2",
     "cordova-plugin-inappbrowser": "3.1.0",
     "cordova-plugin-ionic-keyboard": "2.1.3",
     "cordova-plugin-ionic-webview": "4.1.1",
+    "cordova-plugin-iroot": "npm:@aerogear/cordova-plugin-iroot",
```
Comment (wtrocki) [inline package.json]: nice trick!

**Comment by wtrocki** [comment]
World class job!

**Comment by b1zzu** [comment]
@wtrocki I've added a check to verify that the packege.json doesn't change after cordova build, but it will run only on the master if we think is more important than we should put it somewhere else

also from now on we it will be possible to build ionic-showcase only with cordova > 9, should we document it somewhere?

File: package.json (hunk 1 above)
Comment (b1zzu) [reply package.json]: The only bad thing is that cordova force me to do also this: `"@aerogear/cordova-plugin-iroot": "npm:@aerogear/cordova-plugin-iroot@0.8.1"`

**Pull request merged**